Project.toml

```diff
@@ -0,0 +1,34 @@
+name = "StochDynamicProgramming"
+uuid = "f6024387-75e5-5fdf-b095-327b05c0905f"
+authors = ["mrchaos <mrchaos@naver.com>"]
+version = "0.5.0"
+
+[deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+CutPruners = "65d46eb8-70e9-5a30-bf48-2afa3a021b8f"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
+ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
+JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
+SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
+Nullables = "4d1e1d77-625e-5b40-9113-a560ec7a8ecd"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+SharedArrays = "1a1011a3-84de-559e-8e89-a11a2f7dc383"
+Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+Clp = "e2554f3b-3117-50c0-817c-e040a3ddf72d"
```

Comment:
I would add `Cbc` and `Clp` in `[extras]`

Reply:
I tried to move Cbc and Clp to [extras], but I get an error that package Cbc and Clp cannot be installed during test.

Reply:
Ok, let's keep them there then